src/components/language-selector/_template.njk

```diff
@@ -1,6 +1,6 @@
 {% if params.languages | length <= 3 %}
     {% set alternativeLanguages = params.languages | rejectattr("current") %}
-    <ul class="language-links u-d-no@xs@s">
+    <ul class="language-links u-d-no@xs@m">
```

Comment:
The language selector part of the footer will also need updating to match this although I'm not sure if thats within scope of this PR

Reply:
Yep I'll do that it's just a class update.